h/features/models.py

```diff
@@ -35,6 +35,23 @@
 #
 FEATURES_PENDING_REMOVAL = {}
 
+FEATURE_FEATURECOHORT_TABLE = sa.Table(
+    'feature_featurecohort', Base.metadata,
```

Comment:
For the sake of consistency, let's have both join tables in the same part of the file as one another, and named the same way. That is: `featurecohort_feature` (to go with `featurecohort_user`) rather than `feature_featurecohort`).